experimental/plugins/postgres_storage/src/lib.rs

```diff
@@ -992,10 +1009,24 @@ fn _tag_names_from_json(json: &str) -> Result<Vec<TagName>, WalletStorageError>
 #[cfg(test)]
 mod tests {
     use super::*;
+    use std::env;
     use std::ffi::{CString, CStr};
     use std::{slice, ptr};
     use wql::storage::ENCRYPTED_KEY_LEN;
 
+    // TODO We need init() to be the first test run in order to create the wallets database
+    //#[test]
```

Comment:
is this test attribute supposed to commented out?

Reply:
Yes this test is not necessary, storage initialization is called on each test